### PR TITLE
Bug fixes2

### PR DESCRIPTION
**Pull request by au1994**
@vaidik
@vishesh92

### CODE REVIEW

File: movie.sh [hunk 1]
```diff
@@ -1,6 +1,48 @@
...
+     title=$(curl -s http://www.omdbapi.com/\?i\=\&t\=$search | jq -r '.Title');
+
+     #check if rating is null (if curl fails)
+     if [ "$rating" == "$NULL" ] || [ "$rating" == "$EMPTY" ] || [ "$rating" == "$ZERO" ]
+     then 
+	printf "%-10s  %-30s  %-30s\n" "$NA" "$movie_name" "$title";
+     else 
+	printf "%-10s  %-30s  %-30s\n" "$rating" "$movie_name" "$title";
+     fi
+  done | sort -k1 -rn
```
Comment (vaidik) [inline movie.sh:45]: Nice!

File: movie.sh [hunk 2]
```diff
@@ -1,6 +1,48 @@
...
+     search=$(echo $movie_name | tr ._' ' +)
+     
+     #Curl the api to get the json data
+     rating=$(curl -s http://www.omdbapi.com/\?i\=\&t\=$search | jq -r '.imdbRating');
+     title=$(curl -s http://www.omdbapi.com/\?i\=\&t\=$search | jq -r '.Title');
+
+     #check if rating is null (if curl fails)
+     if [ "$rating" == "$NULL" ] || [ "$rating" == "$EMPTY" ] || [ "$rating" == "$ZERO" ]
+     then 
+	printf "%-10s  %-30s  %-30s\n" "$NA" "$movie_name" "$title";
```
Comment (vaidik) [inline movie.sh:41]: May be you can make a function for this? Notice that this line is not very different from line 43.

**Comment by vaidik** [comment]
@au1994 good work!

File: movie.sh [hunk 3]
```diff
@@ -1,6 +1,48 @@
...
+declare -r EMPTY="";
+declare -r ZERO="0"; 
+
+
+#Print the heading first
+printf "%-10s  %-30s  %-30s\n" "RATING" "MOVIE" "TITLE"
+
+for f in $path/*
+  do
+     #strip initial path value to get the movie name
```
Comment (vishesh92) [inline movie.sh:28]: You have used 5 spaces here.

File: movie.sh [hunk 4]
```diff
@@ -1,6 +1,48 @@
...
+then 
+    echo 'Directory not provided. Searching in current directory'
+    dir='.'
+else 
+    echo 'Input Directory is' $1
+    dir=$1
+fi
+
+#Get absolute path from the input directory
+path=$(find ~/ -name $dir)
```
Comment (vishesh92) [inline movie.sh:13]: I guess this will cause some issues if the specified directory exists at multiple paths.

**Pull request merged**